### ISSUE TITLE
Unstable and crashing

### ISSUE THREAD

**Issue by coreybruce** [nsz32/docklike-plugin#108]
Hi I understand this repo is on hold but the docklike plugin has become unstable where it will be buggy with that other issue I reported before and restarts or crashes alot.

**Comment by SwampRabbit** [comment]
> Hi I understand this repo is on hold but the docklike plugin has become unstable where it will be buggy with that other issue I reported before and restarts or crashes alot.

Then why open the issue?  Please don't open Issues unless the goal is to get something actually fixed.

**Comment by coreybruce** [comment]
> > Hi I understand this repo is on hold but the docklike plugin has become unstable where it will be buggy with that other issue I reported before and restarts or crashes alot.
> 
> Then why open the issue? Please don't open Issues unless the goal is to get something actually fixed.

I opened  the issue because I do obvious want it fixed, what do you even mean exactly?

**Comment by SwampRabbit** [comment]
> > > Hi I understand this repo is on hold but the docklike plugin has become unstable where it will be buggy with that other issue I reported before and restarts or crashes alot.
> > 
> > 
> > Then why open the issue? Please don't open Issues unless the goal is to get something actually fixed.
> 
> I opened the issue because I do obvious want it fixed, what do you even mean exactly?

You opened an Issue on a repo that you know is inactive.  You're opening issues that have no substance, but offering no fixes.  Everyone knows this repo is inactive and unstable, you didn't have to open an issue for it just to say it.

Some people have notifications set up for repos they are working on and get emails when new issues get posted.  Not everyone wants an email for issues that say nothing and offer nothing.  Thats probably one the reasons nsz32 stopped working on this.

**Comment by coreybruce** [comment]
Yeah and from the start of the conversation where eventually they will tell me where/what I can do to debug or even help with the issue. I can't give substance if I don't know where or what to do, that's the job of the developer to work with the user to help resolve the issue and yes the repo is on hold I know that but that doesn't mean I still can't help by reporting issues even if isn't being worked on, they can always come back to it when they are ready to rework on the repo to fix the issues, I know no issue waiting or helping resolve the issues on forks of the repo either. 

There is no need to be arrogant or rude about it, I am just reporting a issue.

**Comment by SwampRabbit** [comment]
Ok do whatever you want

**Comment by nsz32** [comment]
4.16 made me use i3 on a daily basis, it's a nice wm to work on. I still use the plugin on my personal laptop.
I have no unstability nor crashes here using xfce 4.16 on an updated arch.
Please explain your pb.

I'm ok to maintain against natural env/libs upgrades.
I'm not working on the plugin anymore because it suffice for my use and i don't need more features.

**Comment by coreybruce** [comment]
I had issues at the time with the plugin restarting and I still have a issue with some icons not showing up with the correct icon or unable to change the icon when I edit the launcher and keep it as the same pinned application, instead it will just open a new instance of it on docklike instead of being on edited launcher
